airflow/cli/commands/db_command.py

```diff
@@ -207,3 +207,14 @@ def cleanup_tables(args):
         confirm=not args.yes,
         skip_archive=args.skip_archive,
     )
+
+
+@cli_utils.action_cli(check_db=False)
+def export_archived(args):
+    """Exports archived records from metadata database."""
+    if not os.path.exists(args.output_path):
+        raise AirflowException(f"The specified --output-path {args.output_path} does not exist.")
```

Comment:
Probably should be an `isdir` check instead? (Also may be a good idea to mention this should be a directory in the help message and documentation; it took me a minute to figure that out.)

Reply:
moved the check to parser